app/views/teacher_mailer/application_declined.text.erb

```diff
@@ -19,29 +19,24 @@ Thank you for applying for QTS in England. Unfortunately, your application has b
 <% unless @view_object.declined_cannot_reapply? || @view_object.from_ineligible_country? %>
 # What you can do next
 
-You’ll be able to make a new application for QTS once you've addressed the decline reasons we've given. If you reapply for QTS, you’ll not be able to review the details of your previous application.
+You'll be able to make a new application once you've addressed all decline reasons given. If you reapply for QTS, you’ll not be able to review the details of your previous application.
 
-[Apply again](<%= new_teacher_interface_application_form_url %>)
+You may want to explore [other routes to getting QTS](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk).
 
-You may want to [explore other routes to getting QTS](https://www.gov.uk/government/publications/apply-for-qualified-teacher-status-qts-if-you-teach-outside-the-uk/routes-to-qualified-teacher-status-qts-for-teachers-and-those-with-teaching-experience-outside-the-uk).
+[Apply again](<%= new_teacher_interface_application_form_url %>)
 
 <% end %>
-<% if @view_object.from_ineligible_country? %>
-[Sign in](<%= new_teacher_session_url %>) for more information about QTS in England.
-<% else %>
 # Decision review
```

Comment:
The decision review section should only be visible for countries that are eligible, so you'll need to keep in the conditional on `@view_object.from_ineligible_country?`.